nilearn/datasets/atlas.py

```diff
@@ -782,3 +782,73 @@ def fetch_coords_dosenbach_2010(ordered_regions=True):
                   networks=out_csv['network'], description=fdescr)
 
     return Bunch(**params)
+
+
+def fetch_atlas_allen_2011(data_dir=None, url=None, resume=True, verbose=1):
+    """Download and return file names for the resting-state networks atlas
+    published by Allen et al. in 2011. The provided images are in MNI152 space.
+
+    Parameters
+    ----------
+    data_dir: string, optional
+        directory where data should be downloaded and unpacked.
+    url: string, optional
+        url of file to download.
+    resume: bool
+        whether to resumed download of a partly-downloaded file.
+    verbose: int
+        verbosity level (0 means no message).
+
+    Returns
+    -------
+    data: sklearn.datasets.base.Bunch
+        dictionary-like object, keys are:
+        - "all_unthresh_tmaps": T-maps of all 75 unthresholded components.
+        - "rsn_unthresh_tmaps": T-maps of 28 RSNs included in E. Allen et al.
+        - "all_unthresh_rsn_labels": Labels for the 28 RSNs.
+        - "aggregate_ic_comps": Aggregate ICA Components.
+
+    References
+    ----------
+    Licence: unknown
+    E. Allen, et al, "A baseline for the multivariate comparison of resting
+    state networks," Frontiers in Systems Neuroscience, vol. 5, p. 12, 2011.
+    See http://mialab.mrn.org/data/index.html for more information
+    on this dataset.
+    """
+    if url is None:
+        url = "http://mialab.mrn.org/data/hcp/"
+
+    dataset_name = "allen_rsn_2011"
+    keys = ("all_unthresh_tmaps",
+            "rsn_unthresh_tmaps",
+            "aggregate_ic_comps")
+
+    opts = {}
+    files = ["ALL_HC_unthresholded_tmaps.nii",
+             "RSN_HC_unthresholded_tmaps.nii",
+             "rest_hcp_agg__component_ica_.nii"]
+
+    labels = {'Basal Ganglia': [21],
```

Comment:
It looks like labels are not ordered. Isn't it ?

Are these collected from data resource ?